source/client-side-encryption/tests/README.md

```diff
@@ -2816,16 +2816,9 @@ This test is continuation of the case 1 and provides a way to complete inserting
 
 ### 22. Range Explicit Encryption
 
-The Range Explicit Encryption tests require MongoDB server 8.0.0-rc14+ for SERVER-91889 and libmongocrypt containing
-MONGOCRYPT-705.
-
-> [!NOTE]
-> MongoDB Server 8.0 introduced a backwards breaking change to the Queryable Encryption (QE) range protocol: QE Range V2
-> libmongocrypt 1.11.0 is required to use the QE Range V2.
-
-> [!NOTE]
-> MongoDB Server 7.0 introduced a backwards breaking change to the Queryable Encryption (QE) protocol: QEv2.
-> libmongocrypt 1.8.0 is configured to use the QEv2 protocol.
+The Range Explicit Encryption tests utilize Queryable Encryption (QE) range protocol V2 and require MongoDB server
+8.0.0-rc14+ for [SERVER-91889](https://jira.mongodb.org/browse/SERVER-91889) and libmongocrypt 1.11.0+ for
+[MONGOCRYPT-705](https://jira.mongodb.org/browse/MONGOCRYPT-705). The tests must not run against a standalone.
```

Comment:
I'm not sure if the standalone requirement was communicated in some previous iteration of this prose test, but it was done by both [libmongoc](https://github.com/mongodb/mongo-c-driver/blob/8a85033b212dabf140442dd094f636ef2d44b8e5/src/libmongoc/tests/test-mongoc-client-side-encryption.c#L6663) and PHPLIB.